android/build.gradle

```diff
@@ -8,7 +8,7 @@ buildscript {
     ext.assertj_version = '3.22.0'
     ext.mockk_version = '1.12.3'
     ext.gradle_maven_publish = '0.19.0'
-    ext.purchases_version = '5.0.0'
+    ext.purchases_version = '5.1.0'
```

Comment:
I figured I might as well peg it to 5.1.0, even though there's a new method in it that we don't support just yet